hr_attendance_calendar/models/hr_attendance_day.py

```diff
@@ -146,9 +148,9 @@ def _compute_due_hours(self):
 
     @api.multi
     @api.depends('attendance_ids.worked_hours')
-    def _compute_worked_hours(self):
+    def _compute_paid_hours(self):
         for att_day in self.filtered('attendance_ids'):
-            worked_hours = sum(att_day.attendance_ids.mapped('worked_hours'))
+            logged_hours = sum(att_day.attendance_ids.mapped('paid_hours'))
```

Comment:
I think in model `hr.attendance` the field worked_hours is still existing so here it should be `logged_hours = sum(att_day.attendance_ids.mapped('worked_hours'))`